lib/exceptions.rb

```diff
@@ -64,7 +64,7 @@ def to_s
         end.compact.first || "<HOSTNAME>"
         formatted_scopes = Machinery::Ui.internal_scope_list_to_string(@scopes)
 
-        cmd = "#{$0} inspect --extract-files --scope=#{formatted_scopes}"
+        cmd = "#{$0} inspect --extract-files --scope=#{formatted_scopes.delete(" ")}"
```

Comment:
I would rather implement a different method to format the scopes without spaces. Adding spaces just to remove them again doesn't make too much sense.